operations/host_spawn.go

```diff
@@ -941,11 +942,11 @@ Examples:
 			dryRun := c.Bool(dryRunFlagName)
 			remoteIsLocal := c.Bool(remoteIsLocalFlagName)
 
-			if strings.HasSuffix(localPath, string(os.PathSeparator)) && !strings.HasSuffix(remotePath, string(os.PathSeparator)) {
-				remotePath = remotePath + string(os.PathSeparator)
+			if strings.HasSuffix(localPath, "/") && !strings.HasSuffix(remotePath, "/") {
+				remotePath = remotePath + "/"
 			}
-			if strings.HasSuffix(remotePath, string(os.PathSeparator)) && !strings.HasSuffix(localPath, string(os.PathSeparator)) {
-				localPath = localPath + string(os.PathSeparator)
+			if strings.HasSuffix(remotePath, "/") && !strings.HasSuffix(localPath, "/") {
+				localPath = localPath + "/"
```

Comment:
does it make sense to use https://golang.org/pkg/path/

Reply:
It won't really help, because path doesn't have a way of detecting the trailing slash (which is what we're actually interested in) and will always try to remove the trailing slash.